### ISSUE TITLE
USE_DATA_CLASS rule is not triggered on classes without a body

### ISSUE THREAD

**Issue by akuleshov7** [saveourtool/diktat#1444]
The following **main** scenario for `USE_DATA_CLASS` is not processed properly:

```
class ExecutionUpdateDto(
    val id: Long,
    val status: ExecutionStatus,
)
```
